### PR TITLE
Avoid a PHP Notice when `render_block()` is called without attributes.

### PR DESCRIPTION
**Pull request by dd32**
## Description

When `render_block()` is called without block attributes set, `wp_render_elements_supports()` causes a PHP Notice.

Example:
```
<?php echo render_block( array( 'blockName' => 'wporg-learn/workshop-details' ) ); ?>

E_NOTICE: Undefined index: attrs in wp-includes/block-supports/elements.php:20
```

## How has this been tested?

 - Visual testing

## Types of changes
Bug fix

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] I've tested my changes with keyboard and screen readers. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal).

### CODE REVIEW

**Comment by dd32** [comment]
To note, I questioned if the `$parsed_block` should just have the `attrs` field set, but it seems like checking for existence is done elsewhere already:

https://github.com/WordPress/wordpress-develop/blob/master/src/wp-includes/class-wp-block.php#L167-L170

**Review by aristath — APPROVED** [review]
I see no problem with adding an extra check here to avoid issues :+1:

**Comment by youknowriad** [comment]
Looks like there's a unit test failure here.

**Comment by oandregal** [comment]
It seems the error is due to the `WP_Block_Template` class being present both in the WordPress environment (latest `master` from `wordpress-develop`) and the Gutenberg plugin. I can repro locally.

**Comment by oandregal** [comment]
@dd32 this needs to be rebased from `trunk`.

**Comment by oandregal** [comment]
I rebased this PR based on [these instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork).

File: lib/block-supports/elements.php [hunk 1]
```diff
@@ -13,7 +13,10 @@
  * @return string                Filtered block content.
  */
 function gutenberg_render_elements_support( $block_content, $block ) {
-	$link_color = _wp_array_get( $block['attrs'], array( 'style', 'elements', 'link', 'color', 'text' ), null );
+	$link_color = null;
+	if ( ! empty( $block['attrs'] ) ) {
+		$link_color = _wp_array_get( $block_attrs, array( 'style', 'elements', 'link', 'color', 'text' ), null );
```
Comment (oandregal) [inline lib/block-supports/elements.php]:
```suggestion
		$link_color = _wp_array_get( $block['attrs'], array( 'style', 'elements', 'link', 'color', 'text' ), null );
```

**Pull request merged**